docs/general/build/smart-contracts/gas-optimization/methodId.md

```diff
@@ -0,0 +1,29 @@
+# MethodId Optimization
+
+When transactions are executed by the Ethereum Virtual Machine (EVM), the accompanying calldata, which specifies the contract function to be executed, incurs gas fees. These fees are calculated based on the calldata size, with 0 bytes costing 4 gas units and non-0 bytes costing 16 gas units. This pricing structure encourages the efficient use of calldata to reduce transaction costs, especially in contracts with high transaction volumes or complex operations.
+
+The first four bytes of calldata, known as the MethodId, identify the specific contract function to be executed. Optimizing function names to increase the number of 0 bytes in the MethodId can significantly reduce gas costs. This optimization is particularly impactful for frequently called functions, as it directly affects the overall gas efficiency of contract interactions.
```

Comment:
I don't think "significantly" is proper here.

Reply:
I made some updates